src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -293,12 +295,17 @@ export class ZaakViewComponent
     this.dateFieldIconMap.set(
       "einddatumGepland",
       new TextIcon(
-        DateConditionals.provideFormControlValue(
-          this.zaak.einddatum
-            ? DateConditionals.isPreceded
-            : DateConditionals.isExceeded,
-          this.zaak.einddatum ?? "",
-        ),
+        (control: FormControl) => {
+          const plannedDate = control.value;
+          if (!this.zaak.einddatum) {
+            return DateConditionals.isExceeded(plannedDate);
+          } else {
+            return DateConditionals.isExceeded(
+              plannedDate,
+              this.zaak.einddatum,
+            );
+          }
```

Comment:
```suggestion
          return DateConditionals.isExceeded(control.value, this.zaak.einddatum ?? null);
```

or even


```suggestion
          return DateConditionals.isExceeded(control.value, this.zaak.einddatum);
```

would do the same right?